meson.build

```diff
@@ -410,6 +417,8 @@ if build_standalone and get_option('plugin_modem_manager')
   add_project_arguments('-DMM_REQUIRED_VERSION="1.10.0"', language : 'c')
   libqmi_glib = dependency('qmi-glib', version : '>= 1.22.0')
   add_project_arguments('-DQMI_REQUIRED_VERSION="1.23.1"', language : 'c')
+  libmbim_glib = dependency('mbim-glib', version : '>= 1.22.0')
+  add_project_arguments('-DMBIM_REQUIRED_VERSION="1.22.0"', language : 'c')
```

Comment:
shouldn't this only be if you mbim-glib option enabled?

Reply:
do you mean the two lines should be removed here? Just use libmbim optional dependency to control?

Reply:
Well if they're not needed drop em. If they are put them in the other option block.

Reply:
> What is this `MBIM_REQUIRED_VERSION` for? I also wonder what the `QMI_REQUIRED_VERSION` is for?

Sorry, I'm just follow the libqmi to add the libmbim, I don't konw what is `MBIM_REQUIRED_VERSION` for.

And I have a question, does the libmbim need a optional dependency like I added `if get_option('mbim-glib')`?

I think put `libmbim_glib = dependency('mbim-glib', version : '>= 1.22.0')` in `get_option('plugin_modem_manager')` is ok. Is my understanding correct? If not, could you give me some suggestions? Thanks!

Reply:
>      Sorry, I'm just follow the libqmi to add the libmbim, I don't konw what is MBIM_REQUIRED_VERSION for.

Please remove the line adding the MBIM_REQUIRED_VERSION thing, it's not needed. I'm going to suggest a different PR to remove QMI_REQUIRED_VERSION and MM_REQUIRED_VERSION.

Reply:
See https://github.com/fwupd/fwupd/pull/3249